it/v3/endpoints/TriggerBsasControllerISpec.scala

```diff
@@ -87,27 +108,42 @@ class TriggerBsasControllerISpec extends IntegrationBaseSpec {
         ("foreign-property", "foreign-property"),
       ).foreach {
         case (typeOfBusiness, hateoasLinkPath) =>
-          s"any valid request is made with typeOfBusiness: $typeOfBusiness" in new Test {
+          s"any valid request is made with typeOfBusiness: $typeOfBusiness" in new NonTysTest {
 
             override def setupStubs(): StubMapping = {
               AuditStub.audit()
               AuthStub.authorised()
               MtdIdLookupStub.ninoFound(nino)
-              DownstreamStub.onSuccess(DownstreamStub.POST, desUrl, OK, Json.parse(desResponse))
+              DownstreamStub.onSuccess(DownstreamStub.POST, downstreamUri, OK, Json.parse(ifsResponse))
```

Comment:
Would be better if this `ifsResponse` was `downstreamResponse` (see earlier comment)